server/hotrod/src/main/java/org/infinispan/server/hotrod/HotRodServer.java

```diff
@@ -273,7 +273,7 @@ public void rejectedExecution(Runnable r, ThreadPoolExecutor e) {
    protected ExecutorService getExecutor(String threadPrefix) {
       if (this.executor == null || this.executor.isShutdown()) {
          DefaultThreadFactory factory = new DefaultThreadFactory(threadPrefix + "-ServerHandler");
-         int workerThreads = Integer.getInteger(WORKER_THREADS_SYS_PROP, configuration.workerThreads());
```

Comment:
Remove `WORKER_THREADS_SYS_PROP` if it's not used anymore.

Reply:
ok